config.yaml

```diff
@@ -136,3 +136,10 @@ options:
       will use the upstream default.
     type: boolean
     default: null
+  internal-lb:
+    description: |
+      Set to true to ensure that external floating ips are not automatically
+      assigned to k8s services.
+      https://bugs.launchpad.net/charm-openstack-integrator/+bug/1877692
```

Comment:
You don't have to put a bug link here. And the description can be more alighted with the upstream one:
https://github.com/kubernetes/cloud-provider-openstack/blob/003e209d5244d5cefed367a12aeaa8124ec90de4/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md#load-balancer
> internal-lb Determines whether or not to create an internal load balancer (no floating IP) by default. Default: false.